### DISCUSSION TITLE
ABSOLUTE软件的使用疑问

### DISCUSSION THREAD

**Discussion by Yunuuuu** [ShixiangWang/self-study#81] [Q&A] [answered]
您好！ 关于ABSOLUTE的输出，有几列我不太明白它的意思，您能完善一下文档吗，我在biostars上又看到您提的问题，但是下面没人回复！！关于您提到的那几点，ABSOLUTE中copy_ratio的算法您现在知道吗？还有输出中这几列，我不太明白他的意思	seg_sigma	W	copy_ratio	modal_cn	expected_cn	subclonal	cancer_cell_frac	ccf_ci95_low	ccf_ci95_high	hz

**Comment by ShixiangWang** [comment, marked as answer]
ABSOLUTE 不是我写的，你可以查询函数文档、官方文档以及直接联系问下原作者。
